CHANGELOG.md

```diff
@@ -1,5 +1,168 @@
+* jenkins: Resilient VirtualBox VM and test directory cleanup [#5584](https://github.com/kubernetes/minikube/pull/5584)
+* Add option to  configure  dnsDomain in kubeAdm [#5566](https://github.com/kubernetes/minikube/pull/5566)
+* Adjusted Terminal Style Detection [#5508](https://github.com/kubernetes/minikube/pull/5508)
+* Replace BR2_EXTERNAL_MINIKUBE_PATH with PKGDIR. [#5533](https://github.com/kubernetes/minikube/pull/5533)
+* Improve notify test coverage [#5487](https://github.com/kubernetes/minikube/pull/5487)
+* Change systemd unit files perm to 644 [#5492](https://github.com/kubernetes/minikube/pull/5492)
+* lint travis.yml [#5536](https://github.com/kubernetes/minikube/pull/5536)
+* Add ingress-dns addon [#5507](https://github.com/kubernetes/minikube/pull/5507)
+* Fixes image repository flags when using CRI-O and containerd runtime [#5447](https://github.com/kubernetes/minikube/pull/5447)
+* Upgrade nginx ingress controller to 0.26.1 [#5514](https://github.com/kubernetes/minikube/pull/5514)
+* Improve the grammar and annotations  [#5455](https://github.com/kubernetes/minikube/pull/5455)
+* Fix pods not being scheduled when ingress deployment is patched [#5519](https://github.com/kubernetes/minikube/pull/5519)
+* PL translation [#5491](https://github.com/kubernetes/minikube/pull/5491)
+* Fix order of parameters to CurrentContext funcs [#5439](https://github.com/kubernetes/minikube/pull/5439)
+* fr: fix translations of environment & existent [#5483](https://github.com/kubernetes/minikube/pull/5483)
+* Remove single-package constants from constants package [#5430](https://github.com/kubernetes/minikube/pull/5430)
+* Add solution for VERR_VMX_MSR_ALL_VMX_DISABLED [#5460](https://github.com/kubernetes/minikube/pull/5460)
+* Improve reportcard [#5422](https://github.com/kubernetes/minikube/pull/5422)
+* Run travis tests in parallel VMs [#5459](https://github.com/kubernetes/minikube/pull/5459)
+* Add helm-tiller addon [#5363](https://github.com/kubernetes/minikube/pull/5363)
+* registry /  go9p: Correct grammar and spelling in comments [#5443](https://github.com/kubernetes/minikube/pull/5443)
+* Improve out test coverage [#5445](https://github.com/kubernetes/minikube/pull/5445)
+* Fix node InternalIP not matching host-only address [#5427](https://github.com/kubernetes/minikube/pull/5427)
+* Fix crash when deleting the cluster but it doesn't exist [#4980](https://github.com/kubernetes/minikube/pull/4980)
+* Improve storageclass test coverage [#5356](https://github.com/kubernetes/minikube/pull/5356)
+* Remove all functions from constants.go [#5409](https://github.com/kubernetes/minikube/pull/5409)
+* improve test coverage for sshutil package [#5283](https://github.com/kubernetes/minikube/pull/5283)
+* improve test coverage in service pkg [#5258](https://github.com/kubernetes/minikube/pull/5258)
+
+Huge thank you for this release towards our contributors: 
+- Abdulla Bin Mustaqeem
```

Comment:
I'm suspicious of this list: we have 54 PR's and 116 contributors?

Reply:
I did delete a pretty large number of PRs, but I can double check what the script is doing and see if anything's wrong.

Reply:
I think our script may be wrong. This should output the approximate list of contributors:

`git log refs/tags/v1.4.0.. --format=%aN --reverse | sort | uniq -w10 | grep -v "bot\$"`

(26 people, minus some obvious dupes)

Reply:
that does seem a lot more reasonable. fixed.